src/detect-engine-content-inspection.c

```diff
@@ -68,16 +68,6 @@
 /**
  * \brief Run the actual payload match functions
  *
- * The following keywords are inspected:
- * - content, including all the http and dce modified contents
- * - isdataat
- * - pcre
- * - bytejump
- * - bytetest
- * - byte_extract
- * - urilen
- * -
```

Comment:
Were these removed because this was a non-comprehensive list?

Reply:
yeah, slightly incorrect too (function isn't aware of http/dce) and since it's a bit switch statement it's also pretty clear